src/scripts/main.js

```diff
@@ -357,4 +357,23 @@ const people = [
 // eslint-disable-next-line no-console
 console.log(people); // you can remove it
 
-// write your code here
+people.forEach((person) => {
+  const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM before attempting to select it. If it doesn't exist, this will cause a runtime error. Consider adding a check to verify the existence of the table element.